src/tests/suite/daos_rebuild.c

```diff
@@ -1563,98 +1564,113 @@ rebuild_cont_destroy_and_reintegrate(void **state)
+    {"REBUILD15: rebuild multiple tgts", rebuild_multiple_tgts, rebuild_small_sub_setup,
+     rebuild_sub_teardown},
+    {"REBUILD16: disconnect pool during scan", rebuild_tgt_pool_disconnect_in_scan,
+     rebuild_small_sub_setup, rebuild_sub_teardown},
+    {"REBUILD17: disconnect pool during rebuild", rebuild_tgt_pool_disconnect_in_rebuild,
+     rebuild_small_sub_setup, rebuild_sub_teardown},
+    {"REBUILD18: multi-pools rebuild concurrently", multi_pools_rebuild_concurrently,
+     rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD19: rebuild with master change during scan", rebuild_master_change_during_scan,
+     rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD20: rebuild with master change during rebuild", rebuild_master_change_during_rebuild,
+     rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD21: rebuild with master failure", rebuild_master_failure, rebuild_sub_setup,
+     rebuild_sub_teardown},
+    {"REBUILD22: connect pool during scan for offline rebuild",
+     rebuild_offline_pool_connect_in_scan, rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD23: connect pool during rebuild for offline rebuild",
+     rebuild_offline_pool_connect_in_rebuild, rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD24: offline rebuild", rebuild_offline, rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD25: rebuild with two failures", rebuild_multiple_failures, rebuild_sub_setup,
+     rebuild_sub_teardown},
+    {"REBUILD26: rebuild fail all replicas before rebuild",
+     rebuild_fail_all_replicas_before_rebuild, rebuild_sub_rf1_setup, rebuild_sub_teardown},
+    {"REBUILD27: rebuild fail all replicas", rebuild_fail_all_replicas, rebuild_sub_setup,
+     rebuild_sub_teardown},
+    {"REBUILD28: rebuild kill rank during rebuild", rebuild_kill_rank_during_rebuild,
+     rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD29: rebuild kill PS leader during rebuild", rebuild_kill_PS_leader_during_rebuild,
+     rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD30: destroy pool during rebuild failure and retry",
+     rebuild_pool_destroy_during_rebuild_failure, rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD31: reintegrate failure and retry", reintegrate_failure_and_retry, rebuild_sub_setup,
+     rebuild_sub_teardown},
+    {"REBUILD32: kill more ranks than RF, then reintegrate", rebuild_kill_more_RF_ranks,
+     rebuild_sub_setup, rebuild_sub_teardown},
+    {"REBUILD33: delay rebuild and extend", rebuild_delay_and_reintegrate, rebuild_sub_setup,
+     rebuild_sub_teardown},
+    {"REBUILD34: delay rebuild and extend", rebuild_delay_and_extend, rebuild_sub_6nodes_rf1_setup,
+     rebuild_sub_teardown},
+    {"REBUILD35: destroy container then reintegrate", rebuild_cont_destroy_and_reintegrate,
+     rebuild_sub_6nodes_rf1_setup, rebuild_sub_teardown},
+    {"REBUILD36: single engine scan lengthy hang", rebuild_long_scan_hang, rebuild_sub_setup,
+     rebuild_sub_teardown},
```

Comment:
i added another UT for incr reint, may need rebase later if my PR land first.

Reply:
what PR is it that added a test to this file? Latest master shows REBUILD35 is the last test in the list.

Reply:
https://github.com/daos-stack/daos/pull/15782 I can rebase later if your PR land first.